tests/unit/src/test/scala/tests/SuperMethodLspSuite.scala

```diff
@@ -70,7 +84,7 @@ class SuperMethodLspSuite extends BaseLspSuite("gotosupermethod") {
     checkSuperMethod(code)
   }
 
-  test("anonymousclass") {
+  test("anonymous class") {
```

Comment:
nit: we try to avoid spaces in test names to make it easier to  run a single test from the command-line
```suggestion
  test("anonymous-class") {
```

Reply:
`testOnly tests.SuperMethodLspSuite -- --tests "*anonymous class*"` works just fine :)

Reply:
I looked at munit docs and with recommended commands there even with '-' it doesn't work :/

Reply:
anyway all test names changed in this file